src/main/mw.c

```diff
@@ -419,10 +420,23 @@ void processRx(void)
     }
 
     throttleStatus_e throttleStatus = calculateThrottleStatus(&masterConfig.rxConfig, masterConfig.flight3DConfig.deadband3d_throttle);
+    rollPitchStatus_e rollPitchStatus =  calculateRollPitchCenterStatus(&masterConfig.rxConfig);
 
+    /* In airmode Iterm should be prevented to grow when Low thottle and Roll + Pitch Centered. */
+    /* This is needed to prevent Iterm winding on the ground, but keep full stabilisation on 0 throttl in air */
```

Comment:
@ledvinap 
Yes there is a small leak, but it really doesnt matter that much. main idea is to have a good solid protection when on the ground and armed. In air there is nearly never centered stick when doing low throttle manouvres. Even if it is the iterm still stays active, but just not growing.

@hydra 
Center stick option is the only solid case. 
You basically want to have Iterm windup protection when in flight except when on the ground armed. The manual should explain this.
I found that stick is nearly never centered when in air. We are here talking about long inverted manouvres and low throttle acro. Even if the stick is centered it doesnt make that much difference.
I tried different approaches with timers etc, but it looked all ugly and too complex.